src/testRunner/unittests/tsbuild/sample.ts

```diff
@@ -168,14 +168,14 @@ namespace ts {
         });
 
         describe("can detect when and what to rebuild", () => {
-            function initializeWithBuild() {
+            function initializeWithBuild(opts?: BuildOptions) {
                 const fs = projFs.shadow();
                 const host = new fakes.SolutionBuilderHost(fs);
                 const builder = createSolutionBuilder(host, ["/src/tests"], { verbose: true });
                 builder.buildAllProjects();
                 host.clearDiagnostics();
                 tick();
-                builder.resetBuildContext();
+                builder.resetBuildContext(opts ? { ...opts, verbose: true } : undefined);

```

Comment:
why add verbose here? Was this for debugging?

Reply:
That verifies the messages and status of what’s building by outputting it to host